src/components/SignerPanel/SignerPanel.js

```diff
@@ -135,11 +182,31 @@ class SignerPanel extends React.Component {
     }
   }
 
+  handleMsgSign = async () => {
+    const { signatureBag, account, onMsgSignSuccess } = this.props
+
+    this.setState({ status: STATUS_SIGNING_MESSAGE })
+    try {
+      const signatureHash = await this.signMessage(
+        signatureBag.message,
+        account
+      )
+      onMsgSignSuccess(signatureBag)
+      signatureBag.accept(signatureHash)
```

Comment:
```suggestion
      signatureBag.resolve(signatureHash)
```

Based on https://github.com/aragon/aragon.js/pull/276/files/31701f5e183ce8e5dca4785da1d2843bb75f5297#r274301434